### DISCUSSION TITLE
Last update of What?

### DISCUSSION THREAD

**Discussion by dbwebapp** [BitCannaGlobal/WeedWallet#89] [General]
On the following picture you can see "Last update" : 5 month ago . What is this data? is it the last time the validator updated their data such as Description or avatar ? or?

Is it an accurate information to show in the way that some user can think, "Oh this validator updated his node 5 months ago , I choose an other one". It can be tricky to understand this data in the right way, I think.....

![Screenshot 2023-05-25 at 22 06 00](https://github.com/BitCannaGlobal/WeedWallet/assets/96944913/21185296-9021-4c3b-af12-70f5e20c3ab8)

**Comment by TJ-BitCanna** [comment]
This is indeed the last update done to the details of a validator (moniker, details, commission, etc).
But seems best to remove it to avoid confusion.

What would be your recommendation to include on this page?

Replace the "Last update" with the validator details set by the --details flag?
And maybe add the website?

**Reply by dbwebapp** [reply]
website and some social media such as Twitter! User would know if the validator is active posting stuff on twitter for example!

**Reply by TJ-BitCanna** [reply]
Ah yes, but it has to be information already linked to the validator on chain, otherwise we can't pull that information :)

We don't have accounts in the webwallet (and won't create that to avoid centralisation) so it has to be decentralised information we can show.

**Reply by dbwebapp** [reply]
That's make sense...

**Comment by dbwebapp** [comment]
I am just curious. From on-chain data, Would it be possible to  : 
- know if the validator was included on the genesis block
- know since when the selected validator started on the active set 

??

**Reply by TJ-BitCanna** [reply]
I guess it is possible to check the signatures on the genesis block, that should not be a problem.

Checking when a validator entered the active set is a different story, not sure if that is possible.

What are your thoughts?
